static/js/read_receipts.js

```diff
@@ -0,0 +1,57 @@
+import $ from "jquery";
+import SimpleBar from "simplebar";
+
+import render_read_receipts from "../templates/read_receipts.hbs";
+import render_read_receipts_modal from "../templates/read_receipts_modal.hbs";
+
+import * as channel from "./channel";
+import {$t} from "./i18n";
+import * as loading from "./loading";
+import * as message_store from "./message_store";
+import * as overlays from "./overlays";
+import * as people from "./people";
+
+export function show_user_list(message_id) {
+    $("body").append(render_read_receipts_modal());
+    overlays.open_modal("read_receipts_modal", {
+        autoremove: true,
+        micromodal: true,
+        on_show: () => {
+            loading.make_indicator($("#read_receipts_modal .loading_indicator"));
+            channel.get({
+                url: `/json/messages/${message_id}/read_receipts`,
+                idempotent: true,
+                success(data) {
+                    const message_author_id = message_store.get(message_id).sender_id;
+                    // Filter author and current user from read receipts list.
+                    const user_ids = data.user_ids.filter(
+                        (id) => id !== message_author_id && id !== people.my_current_user_id(),
+                    );
```

Comment:
I'm removing this`filter` in the next revision since we are excluding the author in the API itself and [want](https://chat.zulip.org/#narrow/stream/378-api-design/topic/read.20receipts/near/1365286) to display the current user.

Reply:
Yep, thanks!